### PR TITLE
Connector documentation

### PR DESCRIPTION
**Pull request by guiangumpac**
### Description
- Created a markdown file for Power BI certification. Modified and added a few points to https://github.com/Bit-Quill/opensearch-project-sql/blob/main/sql-odbc/docs/user/power_bi_support.md.
- Used https://docs.microsoft.com/en-us/power-query/providing-user-documentation as a guide.
- A known limitations and issues section will be added after analysis of manual tests have finished to have a better idea of the limitations.

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass, including unit test, integration test and doctest
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
  - [ ] New functionality has user manual doc added
- [ ] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: bi-connectors/PowerBIConnector/OpenSearch.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+<img src="img/pbi_select_connector.png" width="500">
+
+3. You will get a warning for using third-party service. Click on **Continue**.
+
+<img src="img/pbi_third_party_warning.png" width="500">
+
+4. Enter server value. Click on **OK**.
+
+<img src="img/pbi_connection_string_options.png" width="500">
```
Comment (Yury-Fridlyand) [inline bi-connectors/PowerBIConnector/OpenSearch.md:37]: As of PR#16 this dialog is changed.

**Review by Yury-Fridlyand — APPROVED** [review]
LGFM

File: bi-connectors/PowerBIConnector/OpenSearch.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+* [OpenSearch SQL ODBC driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/)
+
+## Capabilities supported
+* Import
+* DirectQuery
+
+## Connect to OpenSearch
+1. Open Power BI Desktop.
+
+2. Click on **Home** > **Get Data** > **More** > **Other**. Select **OpenSearch**. Click on **Connect**.
```
Comment (kylepbit) [inline bi-connectors/PowerBIConnector/OpenSearch.md:27]: Should this deal with installing the connector? What about enabling uncertified connectors?

File: bi-connectors/PowerBIConnector/OpenSearch.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+<img src="img/pbi_simple_graph.png">
+
+
+## Troubleshooting 
+
+* If you get an following error, please install [OpenSearch SQL ODBC Driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/).
+
+<img src="img/pbi_error_driver_not_installed.png" width="350">
+
+* If you get an following error,
```
Comment (kylepbit) [inline bi-connectors/PowerBIConnector/OpenSearch.md:60]:
```suggestion
* If you get the following error,

```

File: bi-connectors/PowerBIConnector/OpenSearch.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+7. Click on **Load**.
+
+8. Select required columns for creating graph.
+
+<img src="img/pbi_simple_graph.png">
+
+
+## Troubleshooting 
+
+* If you get an following error, please install [OpenSearch SQL ODBC Driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/).
```
Comment (kylepbit) [inline bi-connectors/PowerBIConnector/OpenSearch.md:56]:
```suggestion
* If you get the following error, please install [OpenSearch SQL ODBC Driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/).

```

File: bi-connectors/PowerBIConnector/OpenSearch.md (hunk 1 above)
Comment (guiangumpac) [reply bi-connectors/PowerBIConnector/OpenSearch.md]: I've updated this screenshot to reflect changes from PR#16

File: bi-connectors/PowerBIConnector/OpenSearch.md (hunk 2 above)
Comment (guiangumpac) [reply bi-connectors/PowerBIConnector/OpenSearch.md]: I don't believe so as the user shouldn't have to install the connector once the connector is certified. For example [Azure](https://github.com/MicrosoftDocs/powerquery-docs/blob/main/powerquery-docs/Connectors/Azure%20Cosmos%20DB%20v2.md) does not include installation of the connector.

File: bi-connectors/PowerBIConnector/OpenSearch.md (hunk 3 above)
Comment (guiangumpac) [reply bi-connectors/PowerBIConnector/OpenSearch.md]: This has now been updated.

File: bi-connectors/PowerBIConnector/OpenSearch.md (hunk 4 above)
Comment (guiangumpac) [reply bi-connectors/PowerBIConnector/OpenSearch.md]: This has now been updated.

File: bi-connectors/PowerBIConnector/OpenSearch.md (hunk 2 above)
Comment (kylepbit) [reply bi-connectors/PowerBIConnector/OpenSearch.md]: Ack - so the intent is this is for the certified connector.

File: bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+* DirectQuery
+
+## Connect to Amazon OpenSearch Service
+1. Open Power BI Desktop.
+
+2. Click on **Home** > **Get Data** > **More** > **Other**. Select **Amazon OpenSearch Service**. Click on **Connect**.
+
+<img src="img/pbi_select_connector_amz.png" width="500">
+
+3. You will get a warning for using third-party service. Click on **Continue**.
```
Comment (kylepbit) [inline bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:30]:
```suggestion
3. You will get a warning for using a third-party service. Click on **Continue**.

```

File: bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
...
+
+<img src="img/pbi_third_party_warning.png" width="500">
+
+4. Enter server value. Click on **OK**.
```
Comment (kylepbit) [inline bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:34]:
```suggestion
4. Enter the server value. Click on **OK**.

```

Comment (kylepbit) [reply bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:34]: What is the server value? The hostname of the OpenSearch service?

File: bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md [hunk 7]
```diff
@@ -0,0 +1,66 @@
...
+
+<img src="img/pbi_auth.png" width="500">
+
+6. Select required table. Data preview will be loaded.
+
+<img src="img/pbi_data_preview.png">
+
+7. Click on **Load**.
+
+8. Select required columns for creating graph.
```
Comment (kylepbit) [inline bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:48]:
```suggestion
8. Select required columns for creating a graph.

```

File: bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+
+<img src="img/pbi_simple_graph.png">
+
+
+## Troubleshooting 
+
+* If you get the following error, please install [OpenSearch SQL ODBC Driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/).
```
Comment (kylepbit) [inline bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:55]:
```suggestion
* If you get the following error, please install the [OpenSearch SQL ODBC Driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/).

```

File: bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+
+<img src="img/pbi_error_driver_not_installed.png" width="350">
+
+* If you get the following error,
+
+<img src="img/pbi_error_conn.png" width="350">
+
+1. Check if host and port values are correct.
```
Comment (kylepbit) [inline bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:63]: Note above that you say the "server" value, but don't mention host and port.

File: bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md (hunk 6 above)
Comment (guiangumpac) [reply bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:34]: This should be updated. This was copied from the [old markdown file](https://github.com/Bit-Quill/opensearch-project-sql/blob/main/sql-odbc/docs/user/power_bi_support.md#:~:text=Enter%20server%20value.%20Click%20on%20OK.). I will update both files

File: bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md (hunk 9 above)
Comment (guiangumpac) [reply bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:63]: Will update this. Thanks

Comment (guiangumpac) [reply bi-connectors/PowerBIConnector/docs/AmazonOpenSearchService.md:63]: Updated the instructions above

File: sql-odbc/docs/user/power_bi_support.md [hunk 10]
```diff
@@ -26,22 +26,23 @@
...
-* Enter server value. Click on **OK**.
+* Enter host and port values and select your preferred SSL option. Click on **OK**.
 
 <img src="img/pbi_connection_string_options.png" width="500">
 
 * Select authentication option. Enter credentials if required and click on **Connect**.
 
 <img src="img/pbi_auth.png" width="500">
 
+
```
Comment (kylepbit) [inline sql-odbc/docs/user/power_bi_support.md:45]: Why the extra newline here?

**Review by kylepbit — APPROVED** [review]

File: sql-odbc/docs/user/power_bi_support.md (hunk 10 above)
Comment (guiangumpac) [reply sql-odbc/docs/user/power_bi_support.md]: Removed.

**Pull request merged**